plasma/plasma-mvp/specifications/morevp.md

```diff
@@ -0,0 +1,648 @@
+
+Additionally, the MoreVP exit protocol is not necessary in all cases.
+Previous work shows that we can use the Plasma MVP exit protocol without confirmation signatures for any transaction included before an invalid (or, in the case of withheld blocks, potentially invalid) transaction.
+We therefore only need to make use of the MoreVP protocol for the set transactions that are in-flight when a Plasma chain becomes byzantine.
+This also means that we can force deposit transactions to use the Plasma MVP exit protocol as these transactions would never be in-flight.
+
+The MoreVP protocol guarantees that if transaction is exitable then either the unspent inputs or unspent outputs can be withdrawn.
+Whether the inputs or outputs can be withdrawn depends on if the transaction is canonical.
+However, in the particular situation in which MoreVP exits are required, users may not be aware that an in-flight transaction is actually non-canonical.
+This can occur if the owner of an input to an in-flight transaction is malicious and has signed a second transaction spending the same input.
+
+To account for this problem, we allow exits to be ambiguous about canonicity.
+Users can start MoreVP exits with the *assumption* that the referenced transaction is canonical.
+Other owners of inputs or outputs to the transaction can then join (or “piggyback”) the exit.
+We add cryptoeconomic mechanisms that determine whether the transaction is canonical and which of the inputs or outputs are unspent.
+The end result is that we can correctly determine which inputs or outputs should be paid out.
+
+
+### MoreVP Exit Protocol Specification
+
+#### Timeline
+
+The MoreVP exit protocol requires the use of a “challenge-response” mechanism, whereby users can submit a challenge but are subject to a response that invalidates the challenge.
+To give users enough time to respond to a challenge, the exit process is split into two “periods.” When challenges are subject to a response, we require that the challenges be submitted before the end of the first exit period and that responses be submitted before the end of the second.
+We define each period to have a length of half the minimum exit period (MEP).
+Currently, the MEP is set to 7 days, so each period has a length of 3.5 days.
+
+
+#### Starting the Exit
+
+For simplicity, we block deposit transactions from using the MoreVP exit protocol.
+
+Any user may initiate an exit by presenting a spend transaction and proving that the transaction is exitable.
+The user must submit a bond, `exit bond`, for starting this action.
+This bond is later used to cover the cost for other users to publish statements about the canonicity of the transaction in question.
+
+We provide several possible mechanisms that allow a user to prove a transaction is exitable.
+First, remember that deposit transactions are exitable by default.
+Two ways in which spend transactions can be proven exitable are as follows:
```

Comment:
use symbols, i.e. `tx` is "the transaction" and `input_tx1, ... input_txn` the other transactions

Reply:
also clarify which signatures are checked to be correct - these that are included in `tx` and pointing towards `newowners'` of `input_txs` or do you go one step further back